scripts/website/generate-sw.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc.
+ * All Rights ReserveThis software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+'use strict';
+
+const workboxBuild = require('workbox-build');
+
+workboxBuild
+  .generateSW({
+    swDest: './dist/website/sw.js',
+    globDirectory: './dist/website/',
+    globPatterns: ['**/*.{js,css,html}'],
```

Comment:
note: this doesn't pick up the _dll javascript files because they are huge.

Reply:
Could you please share what these `_dll` javascript files are? The only reference I make when I hear about `dll` files is the files that we are made at compile-time in `.NET`

Reply:
This is the most I have, what I've seen it seems to be a way to quickly compile dependencies into a file that is faster to build, but not as efficient. https://webpack.js.org/plugins/dll-plugin/